src/ln/channelmonitor.rs

```diff
@@ -226,6 +246,9 @@ pub struct ChannelMonitor {
 	prev_local_signed_commitment_tx: Option<LocalSignedTx>,
 	current_local_signed_commitment_tx: Option<LocalSignedTx>,
 
+	prev_latest_per_commitment_point: Option<SecretKey>,
```

Comment:
I dont think we actually need these right now? They are only used for the local htlc tx case, but I think we actually want the delayed_payment_key there, which we derive.

Reply:
Do we need more than the local_delayedkey anywhere? Maybe only store that?